test/unit/lib/plugins/aws/package/compile/events/event-bridge/index.test.js

```diff
@@ -688,3 +688,42 @@ describe('EventBridgeEvents', () => {
     );
   });
 });
+
+describe('test/unit/lib/plugins/aws/package/compile/events/event-bridge/index.test.js', () => {
+  describe('regular configuration', () => {
+    let customNameEventBridgeResource;
+
+    before(async () => {
+      const { awsNaming, cfTemplate } = await runServerless({
+        fixture: 'function',
+        command: 'package',
+        configExt: {
+          functions: {
+            customName: {
+              handler: 'basic.handler',
+              name: 'custom-name-lambda',
```

Comment:
`runServerless` calls are expensive, so maybe let's add this test case to run that's configured at L590

Reply:
Whoops, fixed!